.github/workflows/maven.yaml

```diff
@@ -300,12 +300,6 @@ jobs:
               -Dskip.build.image=true          
             fi
 
-      - name: Publish Test Report
```

Comment:
this is never executed anyway, meaning it constantly fails. If it failed for so much time and no cared, may be we don't need it at all? :) 

I was working on a different thing on the pipeline and this is how I noticed this.

Reply:
Agree nice to have but if it saves time thats a bonus